Source/WebCore/platform/network/cf/FormDataStreamCFNet.cpp

```diff
@@ -132,8 +132,8 @@ static bool advanceCurrentStream(FormStreamFields* form)
     bool success = WTF::switchOn(nextInput.data,
         [form] (Vector<uint8_t>& bytes) {
             size_t size = bytes.size();
-            MallocPtr<uint8_t, WTF::VectorBufferMalloc> data = bytes.releaseBuffer();
-            form->currentStream = adoptCF(CFReadStreamCreateWithBytesNoCopy(0, data.get(), size, kCFAllocatorNull));
+            auto data = bytes.releaseBuffer();
+            form->currentStream = adoptCF(CFReadStreamCreateWithBytesNoCopy(0, data.span().data(), size, kCFAllocatorNull));
```

Comment:
I’d be tempted to just write out `bytes.releaseBuffer().span().data()` without a local variable.

Reply:
The local variable is used on the next line though.

Reply:
Oops, my mistake!